lib/countly.js

```diff
@@ -2002,6 +2028,66 @@
+        // get enable widgets by app_key
+        // define xhr object
+        
+        var enableWidgets = params.widgets;
+
+        if (enableWidgets.length > 0) {
+            
+            document.body.innerHTML += '<div id="cfbg"></div>';
+
+            var getWidgets = new XMLHttpRequest();
+            getWidgets.open('GET', Countly.url+ '/o/web-feedback/multiple-widgets-by-id?app_key='+Countly.app_key+'&widgets='+JSON.stringify(params.widgets));
+            // prepare response callback 
+            getWidgets.onload = function() {
+                if (getWidgets.status === 200) {
+                    // widget object
+                    var widgets = JSON.parse(getWidgets.responseText);
+                    asyncForeach(widgets, function(widget, done) {
+                        JSON.parse(widget.target_pages).forEach(function(page) {
+                            if (page === window.location.pathname && !widget.hide_sticker) document.body.innerHTML += '<div style="color:'+((widget.trigger_font_color < 7) ? '#'+widget.trigger_font_color : widget.trigger_font_color)+";background-color:"+((widget.trigger_bg_color.length < 7) ? '#'+widget.trigger_bg_color : widget.trigger_bg_color)+'" class="countly-feedback-sticker '+widget.trigger_position+'" id="countly-feedback-sticker-'+widget._id+'">'+widget.trigger_button_text+'</div><div class="countly-iframe-wrapper" id="countly-iframe-wrapper-'+widget._id+'"><span class="countly-feedback-close-icon" id="countly-feedback-close-icon-'+widget._id+'">×</span><iframe name="countly-feedback-iframe" id="countly-feedback-iframe" src="'+Countly.url+"/feedback?widget_id="+widget._id+"&app_key="+Countly.app_key+'&url='+Countly.url+'"></iframe></div>';
+                        });    
+                        JSON.parse(widget.target_pages).forEach(function(page) {
+                            if (page === window.location.pathname && !widget.hide_sticker) {
+                                add_event(document.getElementById('countly-feedback-sticker-'+widget._id), 'click', function(){document.getElementById('countly-iframe-wrapper-'+widget._id).style.display = "block";document.getElementById('cfbg').style.display = "block";});
+                                add_event(document.getElementById('countly-feedback-close-icon-'+widget._id), 'click', function(){document.getElementById('countly-iframe-wrapper-'+widget._id).style.display = "none";document.getElementById('cfbg').style.display = "none";});
+                            }
+                        });
+                        done();
+                    });
+                } else {
+                    console.log('Widget couldn\'t loaded.');
```

Comment:
"couldn't loaded" is funky grammar.

Please consider for example "Widget could not be loaded".